spec/controllers/idv/sessions_controller_spec.rb

```diff
@@ -43,5 +43,14 @@
       expect(flash).to be_empty
       expect(response).to redirect_to(idv_questions_path)
     end
+
+    it 'shows failure on trigger values' do
```

Comment:
What are trigger values?  Maybe I'm just ignorant of maybe the description could be more clear?